### DISCUSSION TITLE
Create a new toaru-ports repository

### DISCUSSION THREAD

**Discussion by mxlgv** [klange/toaruos#235] [Ideas]
I suggest you create the toaru-ports repository. For ease of porting and assembly. But probably for this you will also have to move libc to a separate toaru-libc repository. And use it as a submodule for toaruos and toaru-ports.

**Comment by klange** [comment]
This request doesn't make any sense.

What even is a "ports repository"? When porting software to ToaruOS, it either needs extensive changes (eg. SDL) that warrant a fork with a dedicated repository, or it just needs patches to a `config.sub` script, in which case I'm not going to make a repository just to hold patch files for that.

Further, what is this nonsense about the libc? Why would the libc need to be separated?
